### PR TITLE
human-data_december_hackathon_changes

### PR DESCRIPTION
**Pull request by pinarpink**
Creating a branch to work on the changes to the HD page.

### CODE REVIEW

File: pages/your_domain/human_data.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ---
 title: Human data
-description: Data management solutions for human data.
+description: Key data management considerations solutions for projects with human data.
```
Comment (bedroesb) [inline pages/your_domain/human_data.md:3]: considerations solutions, I think one of the two needs to go :)

Comment (bedroesb) [reply pages/your_domain/human_data.md:3]: @pinarpink What do you think? :)

Comment (pinarpink) [reply pages/your_domain/human_data.md:3]: @bedroesb  yeah, we can drop the "solutions."

Comment (bedroesb) [reply pages/your_domain/human_data.md]: Great!

File: pages/your_domain/human_data.md [hunk 2]
```diff
@@ -91,21 +86,22 @@ For human data, it is very important to use technical and procedural measures to
...
   - [ISO/IEC 27001](https://en.wikipedia.org/wiki/ISO/IEC_27001) is an international information security standard adopted by data centres of some universities and research institutes.
-* Locating tools and platforms suited to handle human data
+* Check whether there are local/national tools and platforms suited to handle human data.
   - Local research infrastructures might have established compute and/or storage solutions with strong information security measures tailored for working on human data (see some examples below). Contact your institute or your ELIXIR node for guidance.
     - Denmark - [Computerome](https://computerome.dk/)
     - Finland - [ePouta](https://research.csc.fi/-/epouta), which is part of the [CSC Tools Assembly](csc_assembly)
     - Norway - [TSD Tools Assembly](tsd_assembly)
     - Sweden - [Bianca](https://www.uppmax.uu.se/resources/systems/the-bianca-cluster/)
     - Spain - [MareNostrum](https://www.bsc.es/marenostrum/access-to-supercomputing-resources)
+    - Luxembourg - [TransMed Tools Assembly](transmed)
```
Comment (bedroesb) [inline pages/your_domain/human_data.md:96]: I am just not sure if we should list national stuff on the domain page ...

Comment (bianchini88) [reply pages/your_domain/human_data.md:96]: I would agree. I think users should be directed towards the national pages where this type of information should be reported (along with national policies).

**Review by bianchini88 — CHANGES_REQUESTED** [review]
* The title "Planning for, and collection of, human data" is not very clear. 
* Spelling: some "ize"s need to become "ise"s (https://rdmkit.elixir-europe.org/style_guide)
*  The reader should be explicitly guided to the national pages on RDMkit (see comment to Bert's comment) and some national-oriented material could probably be removed from this page

**Comment by bedroesb** [comment]
This PR #1075 is also related to this

File: pages/your_domain/human_data.md (hunk 2 above)
Comment (pinarpink) [reply pages/your_domain/human_data.md:96]: Good Idea @bianchini88 I will refer to each national page here.

Comment (pinarpink) [reply pages/your_domain/human_data.md]: I instead referred to the RDMkit national resources index page. As countries other than those in the bulleted list also have resources for sensitive data.

**Comment by bedroesb** [comment]
@bianchini88 would you have a second look if this meets the expectations? thanks!

**Review by bianchini88 — APPROVED** [review]

**Comment by bedroesb** [comment]
Great, we are getting rid of all the PRs! Thanks @pinarpink ;)

**Pull request merged**